README.md

```diff
@@ -32,11 +32,14 @@ import (
   "context"
   "time"
   "github.com/luno/luno-go"
+  "golang.org/x/time/rate"
 )
 
 func main() {
+  rateLimiter := rate.NewLimiter(rate.Every(time.Minute/300), 1)
```

Comment:
Maybe `rateLimiter := rate.NewLimiter(rate.Every(time.Minute), 300)` is more accurate?

Reply:
I tried initially.  What that allows is: a burst of 300 requests and then limits the requests to one per minute.

As I understand it, `rateLimiter := rate.NewLimiter(rate.Every(time.Minute/300), 1)` releases a token to the bucket every 0.2 seconds (60 seconds / 300) and only allows bursts of 1 request at a time.